src/scripts/review.bats

```diff
@@ -187,3 +187,29 @@ setup() {
 		exit 1
 	fi
 }
+
+@test "RC010: All components (jobs, commands, executors, examples) should be snake_cased." {
+	if [[ "${SKIPPED_REVIEW_CHECKS[*]}" =~ "RC010" ]]; then
+		skip
+	fi
+	for i in $(find "${ORB_SOURCE_DIR}jobs" "${ORB_SOURCE_DIR}commands" "${ORB_SOURCE_DIR}executors" -name "*.yml" 2>/dev/null); do
```

Comment:
Highly optional, but we should consider changing the default parameter value not to include the last forward slash. This way, we can do `${ORB_SOURCE_DIR}/jobs` instead of `${ORB_SOURCE_DIR}jobs`. It's more readable.